7.4/php/Dockerfile

```
@@ -0,0 +1,31 @@
+FROM php:7.4-rc-fpm
```

```suggestion
FROM php:7.4.0beta1-fpm
```

Based on the [comment below](https://github.com/WordPress/wpdev-docker-images/pull/1/files#r310881090) using `xdebug-2.8.0beta1` 2.8

Via https://xdebug.org/

> _"This first beta release makes Xdebug compile with the PHP 7.4beta1 release, and also fixes a handful of other bugs."_


I'd suggest also using that cited build of PHP 7.4, what the difference between `7.4.0beta1-fpm` and `7.4-rc-fpm` actually are, I do not know, but at this point in time an RC is not scheduled until 5th September https://wiki.php.net/todo/php74

---

Tests are passing against 7.4snapshot, so it should be fine to use RC.

https://travis-ci.org/xdebug/xdebug/jobs/567376266